pipelines/pipeline_katello_310.yml

```diff
@@ -45,7 +45,7 @@
     foreman_installer_scenario: katello
     foreman_installer_options_internal_use_only:
       - "--disable-system-checks"
-      - "--foreman-admin-password {{ foreman_installer_admin_password }}"
+      - "--foreman-initial-admin-password {{ foreman_installer_admin_password }}"
```

Comment:
You don't have to modify this since we know 3.10 won't use the new parameter. This is true for most versions below. Only where we know it's nightly or could be (e.g., when the user can control the repos via variables) needs to be modified.